js/src/personal/amanda/Amanda.page.tsx

```diff
@@ -1,34 +1,45 @@
-import { Title, Text } from '@mantine/core'
+import { Button, Center, Title } from '@mantine/core'
+import { useViewportSize } from '@mantine/hooks'
+import { useState } from 'react'
 import styles from './Amanda.module.css'
+import { AmandaMainPage } from './Amanda.main.page.tsx'
 
 export function AmandaPage() {
+  const { height, width } = useViewportSize()
+  const [landingPage, setLandingPage] = useState(1)
+
+  // render loading page
+  if (landingPage === 1) {
     return (
-        <>
+      <>
+        <div className={styles.landingPage}>
+          <Center
+            w={width}
+            h={height / 2}
+          >
             <Title
-                className={styles.title}
-                ta="center"
-                mt={100}
+              className={styles.title}
+              ta="center"
+              mt={100}
             >
-                Welcome to{' '}
-                <Text
-                    inherit
-                    variant="gradient"
-                    component="span"
-                    gradient={{ from: 'green', to: 'blue' }}
-                >
-                    Amanda's Page
-                </Text>
+              {"Hi! I'm Amanda!"}
             </Title>
-            <Text
-                c="dimmed"
-                ta="center"
-                size="lg"
-                maw={580}
-                mx="auto"
-                mt="xl"
+          </Center>
+          <Center>
+            <Button
+              bg="none"
+              variant="outline"
+              color="white"
+              onClick={() => setLandingPage(0)}
```

Comment:
It might make sense to update the history and urls with multiple pages like this.